index.bs

```diff
@@ -4116,8 +4116,8 @@ Publishing Registries</h4>
 
 	<ul>
 		<li>
-			Since [=Registry Reports=] are not subject to the [[PATENT-POLICY]],
-			none of their publications correspond,
+			[=Registry reports=] are not subject to the [[PATENT-POLICY]],
+			and therefore none of their publications correspond,
```

Comment:
A Registry definition is a normal subject-to-AC-and-PP recommendation, no?

Reply:
No, only documents on the Recommendation Track are subject to the Patent Policy (per the Patent Policy)... if we want to have the registry definition subject to the Patent Policy, it's going to have to be on the REC track. (We can make special rules about skipping CR, but it can't be on a separate track.)

Reply:
(We probably need to get PSIG review on whether registry definitions should or should not be subject to the Patent Policy, but in any case, if they need to be then they need to be on the REC track, and if they do not need to be, then it's better for them to not be on the REC track.)

Reply:
When I suggested this division (referencing spec., definition, tables) I had the first two on the Rec. Track. The argument that definitions don't need a CR phase for implementation caused the design of the separate track. I don't think we discussed whether that should mean taking it off the Patent Policy as well. I need to think, and the question needs visibility.